libtenzir/builtins/operators/pseudonymize.cpp

```diff
@@ -201,6 +203,65 @@ class plugin final : public virtual operator_plugin<pseudonymize_operator> {
   }
 };
 
+class plugin2 : public virtual function_plugin {
+  auto name() const -> std::string override {
+    return "pan_encode";
```

Comment:
Let's call it `cryptopan_encode` since most folks know this algorithm as CryptoPAn.

Reply:
`cryptopan_encode` vs `crypto_pan_encode`?